src/modules/buttons/Button.jsx

```diff
@@ -6,18 +6,29 @@ import './Button.css';
 
 const propTypes = {
   label: PropTypes.string,
+  mod: PropTypes.string,
   type: PropTypes.string,
+  onClick: PropTypes.func,
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]),
 };
 
 const b = block('j-button');
 
-const Button = (props) => {
+const addMod = mod => mod ? { [mod]: !!mod } : {};
+
+const Button = ({ label, mod, type, onClick, children }) => {
+  const inner = children ? children : label;
```

Comment:
yeah. That's I was talking about. )
Why do we need `<Button label="Something" />` if we can do the same using children